src/sentry/lang/native/symbolicator.py

```diff
@@ -135,10 +140,13 @@ def _process(self, task_name: str, path: str, **kwargs):
                     # Symbolicator instance.
                     session.reset_worker_id()
                     task_id = None
+                    # Backoff on repeated failures to create or query a task.
+                    backoff.sleep_failure()
                     continue
                 finally:
                     self.on_request()
 
+                backoff.reset()
```

Comment:
Do we want to reset the backoff in the `TaskIdNotFound` branch?

Reply:
Yes, that is a successful result from Symbolicator.